objects/vehicle.py

```diff
@@ -0,0 +1,8 @@
+class Vehicle:
+    def __init__(self, make_model, year, color, engine, name, affiliations):
+        self.make_model = make_model
+        self.year = year
+        self.color = color
+        self.engine = engine
+        self.name = name
+        self.affiliations = affiliations
```

Comment:
Affiliation is the header like driver or vehicle, so it needs its own section to put 'sponsor' and 'club' under.

Reply:
We decided to store the affiliations in the Vehicle class, but it is a good idea to split it into sponsors and club